test/extension.test.ts

```diff
@@ -1,8 +1,15 @@
 /* global suite, test */
 
-const vscode = require('vscode');
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
```

Comment:
I prefer explicit exports over defaults, that way incorrect imports are caught early on. It's more of a personal preference. Let's take a call at the end of the PR